Plugin~/WebRTCPlugin/Context.cpp

```diff
@@ -177,6 +184,16 @@ namespace WebRTC
         nvVideoCapturer->FinalizeEncoder();
     }
 
+    bool Context::SetEncoderType(UnityEncoderType type)
```

Comment:
Do we want to support changing the encoder type ?
If so, then we might need to do the initialization process again.

Reply:
we now cannot support this feature.
To avoid misunderstanding, I will remove this method.